getting_started/workflow/assets/import_process.rst

```diff
@@ -54,11 +54,16 @@ Additionally, extra assets will be preset in the hidden res://.import folder:
 
 .. image:: img/asset_workflow5.png
 
-If any of the files present in this folder is erased (or the whole folder), the
-asset or assets will be reimported automatically. As such, committing this folder
-to the version control system is optional. It can save time on
-reimporting time when checking out in another computer, but it takes considerably
-more space and transfer time. Pick your poison!
+Any assets will be reimported automatically when they, or the
```

Comment:
That's confusing, it sounds like the asset will be reimported when it's deleted, but it's only when the corresponding file in the `res://.import/` folder is deleted that the asset will be reimported.

Reply:
I don't understand. Could you or one of the other maintainers edit this so it sounds accurate please?

Reply:
How about: "Assets will be reimported automatically when they are deleted from the res://.import folder (or if the folder itself is deleted)."